src/main/webapp/js/userMapData.json

```diff
@@ -794,6 +925,7 @@
     "Mercer University",
     "Merrimack College",
     "MGH Institute for Health Professions",
+    "MGH Institute of Health Professions",
```

Comment:
This is the same as the entry preceding this, but since it looks like that this version (with "of") is the correct one, you can remove the one before.